src/sentry/runner/initializer.py

```diff
@@ -188,14 +188,15 @@ def configure_structlog():
 
     structlog.configure(**kwargs)
 
-    lvl = os.environ.get('SENTRY_LOG_LEVEL', None)
+    lvl = os.environ.get('SENTRY_LOG_LEVEL', 'INFO')
     if lvl in logging._levelNames:
```

Comment:
Ah, right. That's a good point.

I guess there will also be cases where there is no CLI options. Say, for standalone scripts that use `from sentry.runner import configure; configure()`